modules/colors/platform.ts

```diff
@@ -21,6 +21,7 @@ const TEMP_ANDROID_PLACEHOLDER = PlatformColor('?attr/colorAccent')
 export const label = Platform.select({
 	ios: PlatformColor('label'),
 	android: TEMP_ANDROID_PLACEHOLDER,
+	default: TEMP_ANDROID_PLACEHOLDER,
```

Comment:
This doesn't feel like the right solution because we wouldn't want a fallback on any color definition to begin with; a fallback shouldn't need need to exist in the color definitions but the type returned from `Platform.select` produces a possibly undefined typing.

The alternative I see is to provide a guarantee or type change in `building-hours/lib/color-helpers` when the foreground and background colors are referenced.